exporter/exporter_message/message.go

```diff
@@ -0,0 +1,63 @@
+package exporter_message
```

Comment:
package name is a bit too verbose, `exporter`/`exporter_message`, its best to call it `message` or `messages`, maybe even `types`, or `convert`.
reapting the expoter is just redundant.